decorator_allowed_groups/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2021-Today: GRAP (<http://www.grap.coop/>)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': "Decorator - Allowed Groups",
+    'summary': """
```

Comment:
The summary should be one line. Move the description to the README.

Reply:
Done (set in one line).